src/MenuItem/SplitItem.php

```diff
@@ -129,8 +129,18 @@ public function getRows(MenuStyle $style, bool $selected = false) : array
         return $this->buildRows(
             array_map(function ($index, $item) use ($selected, $length, $style) {
                 $isSelected = $selected && $index === $this->selectedItemIndex;
+
+                if (is_a($item, CheckableItem::class)) {
```

Comment:
Any reason for `is_a` over `instanceof` ?

Reply:
Old habit I'm trying to break. Will use `instanceof` instead.